test/Target/ExportSMTLIB/attributes.mlir

```diff
@@ -0,0 +1,37 @@
+// RUN: circt-translate --export-smtlib %s | FileCheck %s
+// RUN: circt-translate --export-smtlib --smtlibexport-inline-single-use-values %s | FileCheck %s --check-prefix=CHECK-INLINED
+
+smt.solver () : () -> () {
+
+
+  // CHECK: (assert (let (([[V10:.+]] (forall (([[A:.+]] Int) ([[B:.+]] Int))
+  // CHECK:                           (let (([[V11:.+]] (= [[A]] [[B]])))
+  // CHECK:                           [[V11]]))))
+  // CHECK:         [[V10]]))
+
+  // CHECK-INLINED: (assert (forall (([[A:.+]] Int) ([[B:.+]] Int))
+  // CHECK-INLINED:                 (= [[A]] [[B]])))
+  %1 = smt.forall ["a", "b"] {
+  ^bb0(%arg2: !smt.int, %arg3: !smt.int):
+    %2 = smt.eq %arg2, %arg3 : !smt.int
+    smt.yield %2 : !smt.bool
+  }
+  smt.assert %1
```

Comment:
This test already exists in another file. Instead, we could also add a weight attribute here.